### PR TITLE
Issue #13345: Enable Type Name Check Example Test

### PR DESCRIPTION
**Pull request by piyush-1234**
Issue: #13345

### CODE REVIEW

**Review by romani — APPROVED** [review]
Ok to merge

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/typename/Example1.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <module name="TreeWalker">
+    <module name="TypeName"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.naming.typename;
+
+// xdoc section -- start
+class Example1{
```
Comment (nrmancuso) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/typename/Example1.java:12]:
```suggestion
class Example1 {
```

Please do this for all examples.

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/typename/Example1.java (hunk 1 above)
Comment (piyush-1234) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/typename/Example1.java]: done

**Pull request merged**